helix_fhir_client_sdk/fhir_client.py

```diff
@@ -250,20 +296,9 @@ def get(self) -> FhirGetResponse:
             payload: Dict[str, str] = {}
             headers = {"Accept": "application/fhir+json,application/json+fhir"}
 
-            # if we have an auth server url but no access token then get an access token
-            if self._auth_server_url and not self._access_token:
-                assert (
-                    self._login_token
-                ), "login token must be present if auth_server_url is set"
-                self._access_token = self.authenticate(
-                    http=http,
-                    auth_server_url=self._auth_server_url,
-                    auth_scopes=self._auth_scopes,
-                    login_token=self._login_token,
-                )
```

Comment:
why did we remove the auth block for `get` and `merge` ops?
don't we still need it for our external clients, for instance, Medstar and others, so that they can call GET or MERGE ops when they call our FHIR apis?

Reply:
I just moved the logic into the `access_token` property (line 182) to avoid duplicating this code.